peakinvestigator/internal/logger2.py

```diff
@@ -0,0 +1,38 @@
+## -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+from peakinvestigator.internal.progress import *
+import logging
+import time
+import sys
+import os
+
+logging.getLogger('Progress')
+
+class LoggerProgress2(object):
```

Comment:
Shouldn't you be inheriting from `Progress`? Where is it's corresponding `ProgressFactory` implementation?